docs/user/images/plot_images.py

```diff
@@ -41,22 +42,35 @@ def plot_and_save_functions(module, module_name):
     for func_name in func_names:
         func = getattr(module, func_name)
         try:
-            if callable(func) and hasattr(func, "__call__") and not isinstance(func, type):
+            if (
+                callable(func)
+                and hasattr(func, "__call__")
+                and not isinstance(func, type)
+            ):
                 result = func()
                 if isinstance(result, tuple):
                     result = result[0]
                 if isinstance(result, Device):
                     qp(result)
-                    plt.savefig(os.path.join(script_dir, module_name, f"{func_name}.png"))
+                    plt.savefig(
+                        os.path.join(script_dir, module_name, f"{func_name}.png")
+                    )
                     plt.close()
-                    print(f"Info: in module '{module_name}', function '{func_name}' was plotted")
+                    print(
+                        f"Info: in module '{module_name}', function '{func_name}' was plotted"
+                    )
         except Exception as e:
             if "missing" in str(e):
-                print(f"Warning: in module '{module_name}', function '{func_name}' was not plotted: {e}")
+                print(
+                    f"Warning: in module '{module_name}', function '{func_name}' was not plotted: {e}"
+                )
+                print(
+                    f"function {func_name} did not specify default arguments, and will not be plotted"
```

Comment:
what about one single print for warning like:
"Warning: in module '{module_name}', function {func_name} did not specify default arguments, and will not be plotted"?
I fell like its a bit redundant like this. But as you prefer really I don't have a strong opinion on this.